File: isem-finanzas/common/backend/database/Models/Modelos/SuficienciaModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Models.Modelos
{
    public class SuficienciaModel
    {
        public int? id_suficiencia { get; set; }
        public int? id_area { get; set; }
        public string nombre_area { get; set; }
        public string fecha_recepcion { get; set; }
        public string sp { get; set; }
        public string asunto { get; set; }
        public decimal? monto_solicitado { get; set; }
        public decimal? monto_suficiencia { get; set; }
        public decimal? monto_adjudicado { get; set; }
        public string num_contrato { get; set; }
        public string folio { get; set; }
        public string fecha_liberacion { get; set; }
        public string fecha_limite { get; set; }
        public string fecha_turno { get; set; }
        public int? id_estatus { get; set; }
        public string nombre_estatus { get; set; }
        public string observacion { get; set; }
        public string fuentes_financiamiento { get; set; }
        public int? id_usuario { get; set; }

        public int? id_usuario_estatus { get; set; }
        public int? id_tipo_suficiencia { get; set; }


        public int? num_cheque { get; set; }

        public decimal? monto_cheque { get; set; }

        public string banco_expedicion { get; set; }

        public int? asignacion_Presupuesto { get; set; }
        [NotMapped] public int tipoOperacion { get; set; }
    }
}

File: isem-finanzas-web/src/app/@finanzas/@suficiencias/pages/models/suficiencia.model.ts
import { NgbDate } from '@ng-bootstrap/ng-bootstrap';

export class SuficienciaModel {


  id_suficiencia?: number;
  id_partida?: number;
  nombre_partida?: string;
  id_fuente_financiamiento?: number;
  nombre_fuente?: string;
  id_estatus?: number;
  nombre_estatus?: string;
  id_area?: number;
  nombre_area?: string;
  folio?: string;
  asunto?: string;
  monto_solicitado?: number;
  montoFF?: number;
  fecha_limite?: string;
  fecha_recepcionfront?: NgbDate;
  fecha_Limitefront?: NgbDate;
  fecha_liberacionfront?: NgbDate;
  fecha_recepcion?: string;
  fecha_liberacion?: string;
  observacion?: string;
  fuentes_financiamiento?: string;
  fecha_turno?: string;
  fecha_turnadoFront?: NgbDate;
  sp?: string;
  monto_adjudicado?: number;
  num_contrato?: string;
  monto_suficiencia?: number;
  id_usuario?: number;
  id_usuario_estatus?: number;
  id_tipo_suficiencia?: number;
  num_cheque?: number;
  monto_cheque?: number;
  banco_expedicion?: string;
  asignacion_Presupuesto?: number;
}
